package.json

```diff
@@ -7,21 +7,26 @@
     "node": ">= 4"
   },
   "scripts": {
-    "build": "babel src --out-dir ./dist/src",
+    "build:source": "babel src --out-dir ./dist/src",
+    "build:test": "babel test --out-dir ./dist/test --copy-files",
     "clean": "rimraf dist",
-    "copy:test": "node scripts/copy.js",
     "coveralls": "scripts/coveralls.sh",
+    "make": "npm run build:source && npm run build:test",
```

Comment:
Wouldn't `build` make more sense then `make`?

Reply:
Addressed in https://github.com/apiaryio/dredd-transactions/pull/128/commits/606f57c30ae991d3f20359b3d4ba35d1a3d13c00.